spec/controllers/dams_collections_controller_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'spec_helper'
+require 'net/http'
+require 'json'
+
+describe DamsCollectionsController do
+
+  describe "GET osf_api" do
+    before do
+      @unit = DamsUnit.create pid: 'xx48484848', name: "Test Unit", description: "Test Description", code: "tu", uri: "http://example.com/"
+      @provCollection = DamsProvenanceCollection.create(pid: "uu8056206n", visibility: "public")
+      @provCollection.damsMetadata.content = File.new('spec/fixtures/damsProvenanceCollection_osf.rdf.xml').read
+      @provCollection.save!
+      solr_index (@provCollection.pid)   
+    end
+    after do
+      @provCollection.delete
+      @unit.delete
+    end
+
+    it "should return OSF json" do
+      get :osf_api, id: @provCollection.pid 
```

Comment:
@mcritchlow thanks for check it. I just added developer sign in but still got failed. The wired thing is that I ran many times of tests in share_api branch and all got passed, but after merge the develop branch into it the test got failed. So I've checking the recent changes in develop branch which might affect it.